vital/types/bounding_box.h

```diff
@@ -95,6 +95,32 @@ class bounding_box
     m_bbox =  Eigen::AlignedBox< T, 2 >( ul, lr );
   }
 
+  /**
+   * @brief Create default (invalid) box.
+   */
+  bounding_box()
+  {
+    reset();
+  }
+
+  /**
+   * @brief Check to see if the two corner points are valid.
+   *
+   * @return true if the box is valid
+   */
+  bool is_valid() const
+  {
+    return !m_bbox.isEmpty();
+  }
+
+  /**
+   * @brief Reset the bounding box to an initial invalid state.
+   */
+  void reset()
+  {
+    m_bbox = Eigen::AlignedBox< T, 2 >();
```

Comment:
I don't know if it matters, but `m_bbox.setEmpty()` *might* be more efficient. (At best, I believe the compiler would optimize this into the equivalent.)